scripts/e2e/config/kserve/overlay.yaml

```diff
@@ -14,7 +14,7 @@ spec:
             cpu: 1
             memory: 300Mi
           requests:
-            cpu: 25m
+            cpu: 250m
```

Comment:
Can we use `50m`/`100m`? The GitHub runner has limited CPU so it might also affect the e2e test that also requires a lot of CPUt o spawn model pods.